advocacy_docs/pg_extensions/ldap_sync/installing.mdx

```diff
@@ -20,6 +20,10 @@ Before you begin the installation process:
 
   - [Installing EDB Postgres Extended Server](/pge/latest/installing/)
 
+- Install [pg_cron](https://github.com/citusdata/pg_cron)
```

Comment:
We do have pg_cron here at EDB but we don't have any docs for it. I guess this will do until the docs are in place.  I'll file a ticket for that to be done.